.github/actions/build-avr/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM alpine
+
+RUN apk add --update --no-cache curl gcompat
+# libc6-compat advice from here https://githubmemory.com/repo/arduino/arduino-cli/issues/1378
+# but that doesn't work and gives the error described at https://www.gitmemory.com/issue/arduino/Arduino/5691/757103707
+# using gcompat makes the cli and the compiler work
+
+ENV ARDUINO=/Arduino
+ENV ARDUINO_CLI=/Arduino/cli
+ENV ARDUINO_DIRECTORIES_USER=$ARDUINO
+ENV WORKSPACE=${GITHUB_WORKSPACE}
+ENV PATH="${ARDUINO_CLI}/bin:${PATH}"
```

Comment:
If there is value in the path being updated globally, then GitHub prescribes an alternate method for updating the the system `PATH` variable:
https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#adding-a-system-path

Maybe this would eliminate the need for the `install.sh` script?

Reply:
Install.sh isn't really needed anymore if we stick to using docker so it will most likely disappear. It was my first pass, since I don't like putting a lot of script in docker beyond a few lines, but this doesn't play well with GH actions.

Reply:
Let's remove `install.sh`. It already threw me for a loop, and I assume that will be the case for others that follow.